### ISSUE TITLE
ROAM 3 - Making it more obvious when cursor is within snapping radius

### ISSUE THREAD

**Issue by she-weeds** [roam-qgis/Roam#469]
In ROAM 2 the cursor would only show a blue halo when it was snapping to a point. 

In ROAM 3 the blue halo is now constantly present.

I understand this was done for improved visibility. But I think there's a greater downside in that it's harder to tell whether you're within the snapping radius of a vertex. This can have greater consequences if you're relying on geometry to capture values on other fields or do other filtering tasks later. 

So you end up having to jab/squint a little bit more on the field which is a net negative for usability IMO.

Could I recommend disabling the blue halo, or using a different colour halo when the cursor is within snapping radius? Thanks :)

**Comment by she-weeds** [comment]
Just by way of an update this is still an issue with ROAM 3.0.6 regardless of device, this is a screenshot taken from a desktop computer:

![snapping](https://user-images.githubusercontent.com/41975056/94217104-85fbd200-ff24-11ea-8787-1ecb261552c7.gif)

**Comment by NathanW2** [comment]
Yeah I haven't addressed this one yet.  WIll do it in the next release.
Just had to make 3.0.6 low impact release until we have more testing done
on it.

**Comment by she-weeds** [comment]
Thanks Nathan, totally understand the need to get something working out first. Snapping visibility is not a huge deal at this point in the grand scheme of things. Looking forward to testing out subsequent releases, thanks for keeping Roam going all these years.
